charts/rancher-logging/rancher-logging/3.9.000/values.yaml.orig

```diff
@@ -1,82 +0,0 @@
-# Default values for logging-operator.
```

Comment:
I'm not sure why the make sync workflow generated this file, but this could be related to https://github.com/rancher/charts-build-scripts/issues/6. We need to investigate this further.

In future, I think a quick sanity-check workflow would be to use the scripts in https://github.com/aiyengar2/charts-diff/tree/release-v2.5.6 (that I just built) to be able to double check that the assets being released directly match the contents developers pushed to source, except for known changes due to the scripts (e.g. dropping rcVersions).

Based on the contents I generated, it seems like this was the only miss from the release workflow.

Reply:
Would be nice to include that as part of CI, although once it happens its hard for anyone outside the charts-build-scripts devs to fix it.

Reply:
Good idea. Created a blanket issue here to track this effort https://github.com/rancher/charts-build-scripts/issues/21